.github/workflows/ci-test.yml

```diff
@@ -57,7 +57,7 @@ jobs:
         python -m pip install --upgrade pip wheel
         # TODO(b/232490018): Cython need to be installed separately to build pycocotools.
         python -m pip install Cython -c ./test_constraints.txt
-        TFX_DEPENDENCY_SELECTOR=NIGHTLY pip install -c ./test_constraints.txt --extra-index-url https://pypi-nightly.tensorflow.org/simple --pre --editable .[all]
+        TFX_DEPENDENCY_SELECTOR=NIGHTLY pip install -c ./test_constraints.txt --extra-index-url https://pypi-nightly.tensorflow.org/simple --pre --editable .[all] --no-binary :all:
```

Comment:
What does this change do?

Reply:
The idea was to avoid building wheel for `tfx-dev` as it's [not supported/recommended](https://github.com/tensorflow/tfx/blob/55a076c502c7406c96406acfa5122626f2bb81e2/setup.py#L78) for it 

but that doesn't seems to be working. I'll explore alternatives.